internal/api/static/index.html

```diff
@@ -1,14 +1,391 @@
 <!DOCTYPE html>
-<html>
-<head>
-<title>Tx Submit API</title>
-</head>
-<body>
-<p align="center">
-<img src="txsubmit-logo.png" />
-</p>
-<p align="center">
-GitHub: <a href="https://github.com/blinklabs-io/tx-submit-api">https://github.com/blinklabs-io/tx-submit-api</a>
-</p>
-</body>
+<html lang="en" class="bg-black h-full">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Improved Metrics Dashboard</title>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/htmx/1.9.10/htmx.min.js"></script>
+    <script src="https://cdn.tailwindcss.com"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/3.7.1/chart.min.js"></script>
+    <script>
+      tailwind.config = {
+        theme: {
+          extend: {
+            colors: {
+              terminal: {
+                text: '#F0F0F0',
+                bg: '#1E1E1E',
+                header: '#0F3B82',
+              },
+            },
+          },
+        },
+      };
+    </script>
+    <style>
+      .terminal-3d {
+        transform: perspective(1000px);
+        box-shadow: 0 10px 30px -5px rgba(0, 0, 0, 0.5),
+          20px 20px 60px -10px rgba(0, 0, 0, 0.3),
+          -20px -20px 60px -10px rgba(255, 255, 255, 0.1);
+        transition: all 0.3s ease;
+      }
+      .terminal-3d:hover {
+        transform: perspective(1000px) rotateX(0deg) rotateY(0deg);
+      }
+    </style>
+  </head>
+  <body
+    class="bg-gradient-to-br from-gray-900 to-black text-terminal-text font-mono h-full flex items-center justify-center p-4"
+  >
+    <div
+      class="w-full max-w-6xl bg-terminal-bg rounded-lg overflow-hidden terminal-3d"
+    >
+      <div class="bg-terminal-header p-2 flex items-center">
+        <div class="flex space-x-2">
+          <div class="w-3 h-3 rounded-full bg-red-500"></div>
+          <div class="w-3 h-3 rounded-full bg-yellow-500"></div>
+          <div class="w-3 h-3 rounded-full bg-green-500"></div>
+        </div>
+        <div class="flex-grow text-center text-sm">Blink Labs Software</div>
+      </div>
+      <div class="p-6 h-[800px] overflow-y-auto">
+        <div class="text-sm mb-4">
+          Last login: Today<br />
+          Blink Labs Software: tx-submit-api
+        </div>
+        <div
+          id="metrics-container"
+          class="flex items-center flex-wrap font-mono gap-4 whitespace-pre mb-4"
+          hx-get="http://localhost:8081/"
```

Comment:
This should be the request scheme and host, but port 8081. That way it works on remote hosts.